Update _ide-limitations.md (#2256)
semgrep/semgrep-docs: src/components/reference/_ide-limitations.md

> Semgrep's VS Code extension supports the Pro Engine and cross-file analysis. Other IDE scans use Semgrep Community Edition (CE) for its speed with those scans limited to single-file analysis. As a result, you may encounter a higher rate of false positives.

Reviewer: ```suggestion
Semgrep's VS Code extension supports the use of Pro rules and cross-file analysis. Other IDE scans use Semgrep Community Edition (CE) for its speed, and these scans are limited to single-file analysis. As a result, you may encounter a higher rate of false positives.
```
